static/scss/objects/_documentation.scss

```diff
@@ -43,7 +43,7 @@
       .collapsed {
 
         h4 {
-          background: url("../img/icons/accordion-open.svg") no-repeat right center;
+          background: url('https://assets.ubuntu.com/v1/04d2075a-accordion-open.svg') no-repeat right center;
```

Comment:
Can we use `#assets-path` for these please
ie, `url('#{$assets-path}04d2075a-accordion-open.svg');`